.circleci/config.yml

```diff
@@ -41,10 +41,12 @@ workflows:
 jobs:
   test:
     machine:
-      image: << pipeline.parameters.ubuntu_image >> 
+      image: << pipeline.parameters.ubuntu_image >>
     parameters:
       go_version:
         type: string
+    environment:
+      CI_E2E_FILENAME: "fafa8862/rel-nightly"
```

Comment:
It might be helpful to add a note to the [misc README](https://github.com/algorand/indexer/blob/develop/misc/README.md) explaining when this needs to be updated, especially if this env var is expected to be updated on a regular basis.

Reply:
added a note in the readme.